modules/grandpa/src/mock.rs

```diff
@@ -33,6 +33,9 @@ pub type TestNumber = crate::BridgedBlockNumber<TestRuntime, ()>;
 type Block = frame_system::mocking::MockBlock<TestRuntime>;
 type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<TestRuntime>;
 
+pub const MAX_BRIDGED_AUTHORITIES: u32 = 256;
+pub const MAX_HEADER_SIZE: u32 = 65536;
```

Comment:
nit: why not use same values as production?

Reply:
Changed `MAX_BRIDGED_AUTHORITIES`